src/Role/GravityForms.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace P4\MasterTheme\Role;
+
+/**
+ * Abstract Class GravityForms
+ */
+abstract class GravityForms {
+
+	/**
+	 * Add capabilities to user roles
+	 */
+	public static function add_capaibilities_to_user_roles() {
```

Comment:
just a typo in _capa**i**bilities_ `add_capaibilities_to_user_roles` to `add_capabilities_to_user_roles`